janitor/functions.py

```diff
@@ -3815,3 +3816,59 @@ def jitter(
     df[dest_column_name] = result
 
     return df
+
+
+@pf.register_dataframe_method
+def sort_naturally(
+    df: pd.DataFrame, column_name: str, **natsorted_kwargs
+) -> pd.DataFrame:
+    """
+    Sort an DataFrame by a column using "natural" sorting.
+
+    To accomplish this, we do a natural sort
+    on the unique values that are present in the dataframe.
+    Then, we reconstitute the entire dataframe
+    in the naturally sorted order.
+
+    Natural sorting is provided by the Python package natsort_.
+
+    .. _natsort: https://natsort.readthedocs.io/en/master/index.html
+
+    All keyword arguments to natsort should be provided
```

Comment:
Should we put `natsort` in double tick marks here, and in at least another place in this docstring?

Reply:
Good catch! Thanks @hectormz :smile:.